### PR TITLE
Add CONTRIBUTING.md for posts and pages

### LINKED ISSUE #5: Add contributing guide
Add a `CONTRIBUTING.md` for adding new blog posts and pages.

### PR DESCRIPTION
**Pull request by jonmaddock**
Add instructions for adding content. Closes #5.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+# Contributing
+This guide summarises the [Github guide](https://docs.github.com/en/pages/setting-up-a-github-pages-site-with-jekyll/adding-content-to-your-github-pages-site-using-jekyll) for adding new content to a Jekyll site. The "publishing source" is the `master` branch; anything added to `master` will be published automatically. Firstly, please create an issue and branch for the content you wish to add.
```
Comment (johnnonweiler) [inline CONTRIBUTING.md]: I think we should expand on "create an issue and branch".

Perhaps, "create an issue which explains briefly what you are planning to add (and can be used for discussion)".

Do we want people to work on forks?  I think probably not, in which case should we say something about how to request access?

**Review by johnnonweiler — APPROVED** [review]
I think we should add something at the beginning about welcoming contributions from people working on research software around Culham.

I would be happy for this PR to be merged as is (and then further changes could be made by me or Jon in future PRs).

**Comment by johnnonweiler** [comment]
(I forgot to add an important comment: I haven't tested these instructions.)

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,4 +1,6 @@
 # Contributing
+We welcome contributions from people working on research software around Culham.
```
Comment (jonmaddock) [inline CONTRIBUTING.md:2]: @johnnonweiler How about this?

File: CONTRIBUTING.md (hunk 1 above)
Comment (jonmaddock) [reply CONTRIBUTING.md]: Sure. How about using the issue for discussion of the topic and ideas, then the PR for discussion of the actual content?

I'm ashamed to say I don't know what we should do about people requesting access. Is there any way you can do this through github, or is it a case of them sending me an email and then me adding them? This is my first time dealing with an "organisation" account too.

**Comment by jonmaddock** [comment]
> (I forgot to add an important comment: I haven't tested these instructions.)

The instructions are in #6, and are being reviewed by @jholloc.

File: CONTRIBUTING.md (hunk 1 above)
Comment (jonmaddock) [reply CONTRIBUTING.md]: I've just clarified the contributing guidance; I hope that's about right now.

It looks like users need to be added to an organisation; they can't request access through github. Perhaps we should provide a contact email for this. Personal or UKAEA?

Comment (jonmaddock) [reply CONTRIBUTING.md]: Use UKAEA email.

**Comment by johnnonweiler** [comment]
Latest changes look good to me.

**Pull request merged**